src/Shouldly.Tests/ShouldContain/EmptyArrayScenario.cs

```diff
@@ -13,7 +13,8 @@ protected override string ChuckedAWobblyErrorMessage
         {
             get
             {
-                return "new int[0] should contain 1 but does not" +
```

Comment:
We should extract the actual into a variable, that way when looking at these exceptions it makes sense why the actual is printed again?